groups/bsl/bslstl/bslstl_monostate.h

```diff
@@ -0,0 +1,134 @@
+// bslstl_monostate.h                                                 -*-C++-*-
+#ifndef INCLUDED_BSLSTL_MONOSTATE
+#define INCLUDED_BSLSTL_MONOSTATE
+
+#include <bsls_ident.h>
+BSLS_IDENT("$Id: $")
+
+//@PURPOSE: Provide a default constructible alternative for 'bsl::variant'
+//
+//@CLASSES:
+//  bsl::monostate: default constructible 'bsl::variant' first alternative
+//
+//@SEE_ALSO: bslstl_variant
+//
+//@DESCRIPTION: This component provides a 'bsl::monostate' class.  This class
+// can serve as a first alternative type for a 'bsl::variant' to make the
+// 'bsl::variant' type default constructible.
+//
+
+#include <bslscm_version.h>
+
+#include <bsls_keyword.h>
+#include <bsls_libraryfeatures.h>
+
+#ifdef BSLS_LIBRARYFEATURES_HAS_CPP17_BASELINE_LIBRARY
+#include <variant>  // for 'std::monostate'
+#endif              // BSLS_LIBRARYFEATURES_HAS_CPP17_BASELINE_LIBRARY
+
+namespace bsl {
+#ifdef BSLS_LIBRARYFEATURES_HAS_CPP17_BASELINE_LIBRARY
+using std::monostate;
+#else
+                              // ===============
+                              // class monostate
+                              // ===============
+
+class monostate {
+};
+
+// FREE FUNCTIONS
```

Comment:
```suggestion
// FREE OPERATORS
```
See https://bloomberg.github.io/bde/knowledge_base/coding_standards.html#rule-6-9-2